.ci/.e2e-tests.yaml

```diff
@@ -13,53 +13,70 @@ SUITES:
   - suite: "fleet"
     platforms:
       - labels: "arm64"
-      - labels: "ubuntu-18.04"
+      - labels: "ubuntu-20.04"
         provider: "elastic-package"
     scenarios:
       - name: "Fleet"
-        pullRequestFilter: " && ~debian"
         tags: "fleet_mode_agent"
         platforms:
-          - labels: "ubuntu-18.04"
-            provider: "elastic-package"
-          - labels: "windows-2019"
-            provider: "elastic-package"
+          - labels: "arm64"
+          - labels: "amd64"
```

Comment:
Should we use an array struct here?

```yaml
- labels:
   - "arm64"
   - "amd64"
```

Reply:
I think we should replicate changes in this file in the `.ci/.e2e-tests-for-elastic-agent.yaml` file